From: Dmitry Shachnev <mitya57@debian.org>
Date: Thu, 27 Sep 2018 12:56:25 +0300
Subject: Make test_math_compat pass by adding sphinx.ext.mathbase import

If this module is not in sys.modules cache before the chdir() call
in eval_config_file(), Python 2 will try to load sphinx/ext/mathbase.py
file relative to the new working directory, not by absolute path.

This problem does not happen with Python 3.
---
 tests/test_ext_math.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/test_ext_math.py b/tests/test_ext_math.py
index 738ad8c..ea9f8b8 100644
--- a/tests/test_ext_math.py
+++ b/tests/test_ext_math.py
@@ -18,6 +18,7 @@ import pytest
 from docutils import nodes
 
 from sphinx.testing.util import assert_node
+import sphinx.ext.mathbase
 
 
 def has_binary(binary):
